data/service_updates/20240402.yml

```diff
@@ -0,0 +1,21 @@
+---
+title: Manage users within Manage school experience
+summary: |-
+  We've created a new feature that allows you to invite new school experience users directly from within Manage school experience.
+html_content: |-
+  <p>We’ve updated the Manage school experience service
+  so you can now add new users directly on to your school’s account.</p>
+  <h3>New users</h3>
+  <p>When you add a new user to your school profile, they can:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>advertise experience at your school</li>
+    <li>manage bookings and cancellations</li>
+    <li>add other people from your school</li>
+  </ul>
+  <h3>Adding a new user</h3>
+  <p>To add a new user:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>go to the school dashboard in your account</li>
+    <li>go to the Manage users section</li>
+    <li>go to the Manage users section</li>
```

Comment:
the 2nd and 3rd bullet are the same?

Reply:
Dodgy copy and paste from me in VS code - good catch, thank you.